src/test/ui/rustdoc/deny-invalid-doc-attrs.rs

```diff
@@ -0,0 +1,10 @@
+#![feature(no_core)]
+#![no_core]
```

Comment:
Oh oops, this isn't strictly necessary, I just am unable to build libstd locally due to https://github.com/rust-lang/rust/issues/56650.